ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-nonprod.json

```diff
@@ -0,0 +1,338 @@
+[
```

Comment:
Should be include Application_Closed as well to save the work in future? @smitatalawadekar-hmcts  please advise me